src/OrbitGl/App.cpp

```diff
@@ -1093,8 +1120,8 @@ void OrbitApp::StartCapture() {
       enable_introspection, max_local_marker_depth_per_command_buffer, collect_memory_info,
       memory_sampling_period_ns, std::move(capture_event_processor));
 
-  // TODO (187250643): Refactor this to be more readable and maybe remove parts that are not needed
-  // here (capture cancelled)
+  // TODO (http://b/187250643): Refactor this to be more readable and maybe remove parts that are
```

Comment:
I don't think you need the `http://`.

Reply:
My personal preference to have http:// everywhere it is implied. Easier for tools to recognize that this is a link.

Reply:
Sorry for that. Forgot to git add before amending yesterday. Also fixed it in https://github.com/google/orbit/pull/2291

Reply:
Do you have specific tools that recognize it with `http://` and not without? The styleguide says `b/buganizer_id` and being consistent across our TODOs doesn't hurt.

Reply:
vscode does not recognize it without the http:// 

I agree consistency would be nice. I will take this to the chat room and can then take care of the refactor for the whole codebase.